src/libraries/Microsoft.Extensions.DependencyInjection/src/Microsoft.Extensions.DependencyInjection.csproj

```diff
@@ -10,13 +10,14 @@
     <PackageDescription>Default implementation of dependency injection for Microsoft.Extensions.DependencyInjection.</PackageDescription>
     <!-- Use targeting pack references instead of granular ones in the project file. -->
     <DisableImplicitAssemblyReferences>false</DisableImplicitAssemblyReferences>
+    <ServicingVersion>1</ServicingVersion>
```

Comment:
You also need

```xml
<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Reply:
According to the [doc](https://github.com/dotnet/runtime/blob/main/docs/project/library-servicing.md#check-if-a-package-is-generated) this is needed only if the package marked with `<IsPackable>true</IsPackable>` only. Do you think we still need to include `GeneratePackageOnBuild`?

Reply:
All MS.Extensions libraries are packable. In 6.0 we had it globally:

https://github.com/dotnet/runtime/blob/531f715f281e7a6456375635c02b6dd301d95996/src/libraries/Directory.Build.props#L79

This changed with https://github.com/dotnet/runtime/commit/4d97cda06439540cbc8bdffcd191c7f6184257cd to move it to each project.

Reply:
This information should be documented in the doc @tarekgh pointed out.

Reply:
Actually, that's the `main` doc. The document that should mention the Microsoft.Extensions global IsPackable is this one:

https://github.com/dotnet/runtime/blob/release/6.0/docs/project/library-servicing.md#check-if-a-package-is-generated